frontend/amundsen_application/static/js/components/Badges/BadgeBrowseList/styles.scss

```diff
@@ -10,8 +10,9 @@
   .badge-list {
     display: block;
     font-weight: $font-weight-body-regular;
-    margin-top: $spacer-3;
+    margin-top: $spacer-2;
     margin-bottom: $spacer-2;
+    line-height: 2em;
```

Comment:
I would prefer not to use ems here, as we haven't really used them in other places.
Maybe there is a value related to the spacer that works for this case?